.github/workflows/vapoursynth.yml

```diff
@@ -0,0 +1,151 @@
+      run: |
+        build/script.sh
+    - name: Install VapourSynth on Ubuntu 32-bit
+      if: matrix.toolchain == 'stable-i686-unknown-linux-gnu'
+      run: |
+        build/script.sh --build=i686-linux-gnu \
+                        "CFLAGS=-m32" "CXXFLAGS=-m32" "LDFLAGS=-m32"
+    - name: Install stable
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: ${{ matrix.toolchain }}
+        override: true
+    - name: Run tests
+      run: |
+        python3 build/run-tests.py
+    - name: Run sample-plugin
+      env:
+        FUNCS: vapoursynth/vapoursynth-functions vapoursynth/vsscript-functions
+      run: |
+        cd sample-plugin
+        cargo build --verbose
+        cargo run --verbose --bin test --features cfg-if $FUNCS
+    - name: Run doc
+      uses: actions-rs/cargo@v1
+      with:
+        command: doc
+        args: --all-features --verbose
+    - name: Copy index into the target directory
+      if: matrix.toolchain == 'stable-x86_64-unknown-linux-gnu'
+      run: |
+        cp build/index.html target/doc
+    - name: VapourSynth deployment
+      if: matrix.toolchain == 'stable-x86_64-unknown-linux-gnu'
+      uses: JamesIves/github-pages-deploy-action@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        BASE_BRANCH: master
+        BRANCH: gh-pages
+        FOLDER: target/doc
+
+  windows-tests:
+    strategy:
+      matrix:
+        toolchain:
+         - stable-x86_64-pc-windows-msvc
+         - stable-x86_64-pc-windows-gnu
+         - stable-i686-pc-windows-msvc
+         - stable-i686-pc-windows-gnu
+        include:
+         - toolchain: stable-x86_64-pc-windows-msvc
+           arch: x86_64
+         - toolchain: stable-x86_64-pc-windows-gnu
+           arch: x86_64
+         - toolchain: stable-i686-pc-windows-msvc
+           arch: i686
+         - toolchain: stable-i686-pc-windows-gnu
+           arch: i686
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install VapourSynth for Windows 64-bit
+      if: matrix.arch == 'x86_64'
+      run: |
+        build/windows-script.ps1
+    - name: Install VapourSynth for Windows 32-bit
+      if: matrix.arch == 'i686'
+      run: |
+        build/windows-script.ps1 -arch i686
+    - name: Install stable
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: ${{ matrix.toolchain }}
+        override: true
+    - name: Run tests
+      run: |
+        python build/run-tests.py
+    - name: Build sample-plugin
+      run: |
+        $Env:Path += ";C:\Program Files\VapourSynth;"
+        cd sample-plugin
+        cargo build --verbose
+    - name: Run sample-plugin
+      # https://github.com/rust-lang/rust/issues/50176
+      if: matrix.toolchain != 'stable-i686-pc-windows-gnu'
+      env:
+        FUNCS: vapoursynth/vapoursynth-functions vapoursynth/vsscript-functions
```

Comment:
Same comment about the environment variable.

Reply:
Removed them, thanks!